src/jsn.erl

```diff
@@ -23,6 +23,8 @@
     path_transform/2,
     path_elements/1
 ]).
+%% select API
+-export([select/2, select/3]).
```

Comment:
Picky function placement detail: I don't think that this set of functions needs its own category. I think that these functions should come after `find/N` in the `lookup/storage API` section above. Can you also relocate the implementation (of the public functions) to be in that place as well?